sys/net/gnrc/network_layer/ipv6/ext/gnrc_ipv6_ext.c

```diff
@@ -26,6 +26,38 @@
 #include "debug.h"
 
 #ifdef MODULE_GNRC_IPV6_EXT_RH
+/* unchecked precondition: hdr is gnrc_pktsnip_t::data of the
+ * GNRC_NETTYPE_IPV6 snip within pkt */
+static void _forward_pkt(gnrc_pktsnip_t *pkt, ipv6_hdr_t *hdr)
+{
+    gnrc_pktsnip_t *netif_snip;
+
+    if (--(hdr->hl) == 0) {
+        DEBUG("ipv6_ext_rh: hop limit reached 0: drop packet\n");
+        gnrc_pktbuf_release(pkt);
```

Comment:
no return here?

Reply:
Good catch! (I should have written a test for this in #10392 -.-).

Reply:
Fixed

Reply:
(and added test case to #10392)

Reply:
((and added the sending of the expected error message to https://github.com/RIOT-OS/RIOT/pull/10242))